src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -88,5 +89,13 @@ public static void UpdateHeight(this FrameworkElement nativeView, IView view)
 			// WinUI uses NaN for "unspecified"
 			nativeView.Height = view.Height >= 0 ? view.Height : double.NaN;
 		}
+
+		public static bool TryMoveFocus(this FrameworkElement nativeView, FocusNavigationDirection direction)
+		{
+			if (nativeView?.XamlRoot?.Content is not UIElement xamlRootContent)
+				return false;
+
+			return FocusManager.TryMoveFocus(direction, new FindNextElementOptions { SearchRoot = xamlRootContent });
```

Comment:
We now have to pass in the `FindNextElementOptions` which breaks the focus on scroll views.